internal/gcs/gcs_upload.go

```diff
@@ -44,22 +46,30 @@ func (u *GCSUpload) UploadFile(filename string, contents []byte) error {
 		return errors.New("please initialise the connection")
 	}
 	bucket := viper.GetString("BUCKET_NAME")
-	log.WithField("filename", filename).WithField("bucket", bucket).Info("uploading to bucket")
+	prefix := viper.GetString("PREFIX_NAME")
```

Comment:
This code is repeated twice, might make a good candidate to be refactored into a function and shared between them?

Then we could have a unit test for it too :)

Reply:
Both done.